apps/pkeyutl.c

```diff
@@ -244,6 +247,7 @@ int pkeyutl_main(int argc, char **argv)
     argc = opt_num_rest();
     if (argc != 0)
         goto opthelp;
+    libctx = app_libctx();
```

Comment:
Where is `app_libctx()` defined, let alone declared?

Reply:
My initial comment described the dependancy on #11884.

Reply:
Ok I have added a revised libctx back in, to try to progress this forward.